docs/errors/usage_errors.md

```diff
@@ -365,6 +365,62 @@ class Model(BaseModel):
 assert Model(pet={'pet_type': 'kitten'}).pet.pet_type == 'cat'
 ```
 
+## Callable discriminator no tag {#callable-discriminator-no-tag}
```

Comment:
```suggestion
## Callable discriminator case with no tag {#callable-discriminator-no-tag}
```
The current name of this section feels a little too far from a sensible english statement. Looking briefly at some of the other section titles, even if the title isn't a viable sentence, seem to be like, a more free-standing-capable phrase, if that makes sense.